contrib/git-lfs/template.py

```diff
@@ -0,0 +1,36 @@
+pkgname = "git-lfs"
+pkgver = "3.5.1"
+pkgrel = 0
+build_style = "go"
+make_cmd = "gmake"
+make_build_args = [
+    "-ldflags",
+    "-X github.com/git-lfs/git-lfs/v3/config.Vendor=ChimeraLinux",
+]
+hostmakedepends = ["asciidoctor", "go"]
+checkdepends = ["bash", "git", "gmake"]
+depends = ["git"]
+pkgdesc = "Git extension for versioning large files"
+maintainer = "Subhaditya Nath <sn03.general@gmail.com>"
+license = "MIT"
+url = "https://git-lfs.com"
+source = f"https://github.com/git-lfs/git-lfs/releases/download/v{pkgver}/git-lfs-v{pkgver}.tar.gz"
+sha256 = "fc19c7316e80a6ef674aa4e1863561c1263cd4ce0588b9989e4be9461664d752"
+
+
+def post_build(self):
+    self.log("Building man pages")
```

Comment:
don't need this
```suggestion
    self.log("Building man pages")
```

Reply:
Ah yes, I added it before the `*[for ...]` optimization was added, because that step was taking a LOT longer.

Removed :+1: